File: data_structures/linear/static/array/array.py
# We create an array named my_array with initial values [1, 2, 3, 4, 5].
# We access and print individual elements using indexing.
# We modify the third element of the array to be 10.
# We append a new element 6 to the end of the array.
# We remove the second element from the array using the pop method and store the removed element in the variable removed_element.
# We determine the length of the array using the len function.
# Finally, we print the modified array, the removed element, and the length of the array.

# Creating an array
my_array = [1, 2, 3, 4, 5]

# Accessing elements of the array
print("First element:", my_array[0])
print("Second element:", my_array[1])

# Modifying an element of the array
my_array[2] = 10

# Adding an element to the end of the array
my_array.append(6)

# Removing an element from the array
removed_element = my_array.pop(1)

# Length of the array
array_length = len(my_array)

# Printing the array and its properties
print("Modified Array:", my_array)
print("Removed Element:", removed_element)
print("Length of the Array:", array_length)

File: algorithms/1 data_structures/linear/static/array/array.cs
// The List<int> class is used to represent a dynamic array of integers.
// Elements are accessed using indexing, modified in place, and appended using the Add method.
// An element is removed using RemoveAt based on the index.
// The length of the array is obtained using the Count property.
// The modified array and its properties are then printed.

using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        // Creating an array
        List<int> myArray = new List<int> { 1, 2, 3, 4, 5 };

        // Accessing elements of the array
        Console.WriteLine("First element: " + myArray[0]);
        Console.WriteLine("Second element: " + myArray[1]);

        // Modifying an element of the array
        myArray[2] = 10;

        // Adding an element to the end of the array
        myArray.Add(6);

        // Removing an element from the array
        int removedElement = myArray[1];
        myArray.RemoveAt(1);

        // Length of the array
        int arrayLength = myArray.Count;

        // Printing the array and its properties
        Console.Write("Modified Array: ");
        foreach (int element in myArray)
        {
            Console.Write(element + " ");
        }
        Console.WriteLine();
        
        Console.WriteLine("Removed Element: " + removedElement);
        Console.WriteLine("Length of the Array: " + arrayLength);
    }
}
